### PR TITLE
Updates for fid related changes and added project_parent module

### PR DESCRIPTION
**Pull request by ByteMeAsap**
Updates for invoice mail template,Digest email configuration , added project_parent module and changes for bank statement view

### CODE REVIEW

**Review by ArneDeGeeter — APPROVED** [review]
Looks good to me

File: account_configuration/data/mail_template_data.xml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" ?>
+<odoo>
+    <data noupdate="1">
```
Comment (espo-tony) [inline account_configuration/data/mail_template_data.xml:3]: @ByteMeAsap 
Not sure about the effect of noupdate="1" , considering that you run, within this block, a function to temporarily remove the noupdate attribute. Probably it has no effect.

File: account_configuration/data/mail_template_data.xml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            <value eval="{'noupdate': False}"/>
+        </function>
+        <!--Email template -->
+        <record id="account.email_template_edi_invoice" model="mail.template">
+            <field name="body_html" type="html">
+<div style="margin: 0px; padding: 0px;">
+    <p style="margin: 0px; padding: 0px; font-size: 13px;">
+        Dear
+        <t t-if="object.partner_id.parent_id">
+            <t t-out="object.partner_id.name or ''">Brandon Freeman</t> (<t t-out="object.partner_id.parent_id.name or ''">Azure Interior</t>),
```
Comment (espo-tony) [inline account_configuration/data/mail_template_data.xml:17]: @ByteMeAsap 
Commenting here, but it refers also to similar instances later in this file:

Why are you using `t-out` here? You could use `t-field`, and also get rid of the `or ''` statement.

**Review by espo-tony — COMMENTED** [review]
@ByteMeAsap . Some small remarks

File: account_configuration/data/mail_template_data.xml (hunk 1 above)
Comment (ByteMeAsap) [reply account_configuration/data/mail_template_data.xml:3]: @espo-tony , noupdate="1" would make sure that the change that I am applying through the piece of code would only be applied once when the module("account_configuration") is installed. The temporary removal of the noupdate attribute is for base module ("account") data not for this module ("account_configuration")

File: account_configuration/data/mail_template_data.xml (hunk 2 above)
Comment (ByteMeAsap) [reply account_configuration/data/mail_template_data.xml:17]: I did like the way it is as it was the same in the base odoo module("account") as well. Didn't want to change it completely

**Pull request merged**